docs/src/administration/web/_index.md

```diff
@@ -88,7 +88,7 @@ It is good practice to take a backup of your environment before performing a syn
 
 #### Backup
 
-Creating a backup for an environment means saving a copy of the database so that it can be restored. You will see the backup in the activity feed of you environment in the Platform.sh management console where you can trigger the restore by clicking on the `restore` link.
+Creating a backup for an environment means saving a copy of the environment so that it can be restored. You will see the backup in the activity feed of you environment in the Platform.sh management console where you can trigger the restore by clicking on the `restore` link. For more information please refer [here](/administration/backup-and-restore.md).
```

Comment:
```suggestion
Creating a backup for an environment means saving a copy of the environment so that it can be restored. You will see the backup in the activity feed of your environment in the Platform.sh management console where you can trigger the restore by clicking on the `restore` link. For more information please refer [here](/administration/backup-and-restore.md).
```